projects/aca-content/src/lib/components/header-actions/header-actions.component.html

```diff
@@ -0,0 +1,48 @@
+<div>
+  <div class="adf-toolbar--spacer adf-toolbar-divider"></div>
+  <span class="action-bar">
+    <button class="aca-mat-button aca-create-button" mat-stroked-button data-automation-id="create-button"
+      *ngIf=" isPersonalFilesRoute() || isFavoriteLibrariesRoute() || isLibrariesrRoute() "
```

Comment:
please introduce a single "canShowCreateButton(): boolean" and evaluate all conditions there... you should also have better support to unit test that.... please ensure you have unit tests for this too